shell/platform/darwin/ios/BUILD.gn

```diff
@@ -208,6 +208,7 @@ source_set("ios_test_flutter_mrc") {
     "framework/Source/FlutterPlatformPluginTest.mm",
     "framework/Source/FlutterPlatformViewsTest.mm",
     "framework/Source/FlutterViewTest.mm",
+    "framework/Source/VsyncWaiterIosTest.mm",
```

Comment:
Why does this test need to use mrc? Can it be moved to `ios_test_flutter`?  Your test as written isn't releasing everything.

Reply:
I got compiling issue trying to use ARC due to `scoped_nsobject` used in the header. `"flutter/shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.h"`